ci/pipeline.yml

```diff
@@ -0,0 +1,119 @@
+---
+resource_types:
+  - name: gcs-resource
+    type: docker-image
+    source:
+      repository: frodenas/gcs-resource
+
+resources:
+  - name: gcp-service-broker
+    type: git
+    source:
+      uri: https://github.com/GoogleCloudPlatform/gcp-service-broker.git
+      branch: concourse
+      ignore_paths:
+        - ci/*
+
+  - name: gcp-service-broker-ci
+    type: git
+    source:
+      uri: https://github.com/GoogleCloudPlatform/gcp-service-broker.git
+      branch: concourse
+      paths:
+        - ci/*
+
+  - name: release
+    type: gcs-resource
+    source:
+      bucket: gcp-service-broker-tiles
+      json_key: {{STORAGE_BUCKET_KEY}}
+      regexp: gcp-service-broker-(.*).pivotal
+
+  - name: version
+    type: semver
+    source:
+      bucket: gcp-service-broker-semver
+      key: semver
+      endpoint: storage.googleapis.com
+      access_key_id: {{storage_key_id}}
+      secret_access_key: {{storage_secret_key}}
+      initial_version: 3.1.3
+
+jobs:
+  - name: test-unit
+    plan:
+      - aggregate:
+        - get: gcp-service-broker
```

Comment:
This is aggregating the two resources that are the same git repo. Is that necessary?